requirements/ci-wheel.txt

```diff
@@ -21,7 +22,6 @@ twine==1.11.0
 yarl==1.1.1
 
 # Using PEP 508 env markers to control dependency on runtimes:
-aiodns==1.1.1; platform_system!="Windows"  # required c-ares will not build on windows
```

Comment:
Why are you decommissioning it?

Reply:
aiodns **does** work on Windows (at least appveyor tests are passed)

Reply:
But why remove the whole of it, not just conditional?